spring-batch-docs/asciidoc/step.adoc

```diff
@@ -2018,18 +2022,18 @@ constructs, as shown in the following example:
 <bean id="flatFileItemReader"
       class="org.springframework.batch.item.file.FlatFileItemReader">
     <property name="resource"
-              value="file://outputs/20070122.testStream.CustomerReportStep.TEMP.txt" />
+              value="file://outputs/file.txt" />
 </bean>
 ----
 
 .Java Configuration
 [source, java, role="javaContent"]
 ----
 @Bean
-public FlatFileItemReaer flatFileItemReader() {
+public FlatFileItemReader flatFileItemReader() {
 	FlatFileItemReader<Foo> reader = new FlatFileItemReaderBuilder<Foo>()
 			.name("flatFileItemReader")
-			.resource(new FileSystemResource("file://outputs/20070122.testStream.CustomerReportStep.TEMP.txt"))
```

Comment:
This spans two lines. Since it is just an example, I used a shorter file name so that is rendered better.